src/core/lv_obj_pos.c

```diff
@@ -854,6 +855,15 @@ void lv_obj_invalidate(const lv_obj_t * obj)
     lv_obj_invalidate_area(obj, &obj_coords);
 }
 
+bool is_transformed(const lv_obj_t * obj)
```

Comment:
```suggestion
static bool is_transformed(const lv_obj_t * obj)
```

And move to the `STATIC FUNCTIONS` section at the bottom.

Reply:
Good!. done.

Reply:
Thanks. Could you please also move the function body below the

```c
/**********************
 *   STATIC FUNCTIONS
 **********************/
```

section?

Reply:
yes. thanks for checking